crowbar_framework/app/constraints/api_constraint.rb

```diff
@@ -15,13 +15,14 @@
 #
 
 class ApiConstraint
-  attr_reader :version
+  attr_reader :major, :minor
 
   def initialize(options)
```

Comment:
Maybe let's use kwargs here? Also it will allow us to state 1.0 as an obvious default:

```ruby
def initialize(major: 1, minor: 0)
  @major = major
  @minor = minor
end
```